samples/graph-app-installation-lifecycle/csharp/AppInstallation/Views/Home/Index.cshtml

```diff
@@ -86,7 +86,8 @@
 
 <script type="text/javascript">
     $(document).ready(function () {
-        microsoftTeams.app.initialize();
+        microsoftTeams.app.initialize().then(() => {
```

Comment:
please try once with await microsoftTeams.app.initialize() and make the function async